code/game/turfs/closed/minerals.dm

```diff
@@ -32,9 +32,11 @@
 	var/obj/item/boulder/spawned_boulder = null
 	/// How much ore we spawn when we're mining a mineralType.
 	var/mineralAmt = 3
-	///Holder for the image we display when we're pinged by a mining scanner
+	/// The icon of the image we display when we're pinged by a mining scanner. If unset this will just be the default 'icons/effects/ore_visuals.dmi'
+	var/scan_icon
```

Comment:
Rather than having behavior for being unset, why not set it to `'icons/effects/ore_visuals.dmi'`

Reply:
Correct, lists are the only exception